ci_framework/roles/devscripts/tasks/04_overlays.yml

```diff
@@ -0,0 +1,57 @@
+---
+- name: Get OCP related nodes
+  register: _current_vms
+  community.libvirt.virt:
+    command: "list_vms"
+    uri: "qemu:///system"
```

Comment:
I think it is a fair assumption to make that only the virtual OCP nodes are running at this point of time. Hence we could add `state: running`

wdyt?

Reply:
Allows for co-existence... I think it is good to check than assume.